src/test/java/wolox/training/repositories/BookRepositoryTest.java

```diff
@@ -0,0 +1,51 @@
+package wolox.training.repositories;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import wolox.training.models.Book;
+
+@DataJpaTest
+@ExtendWith(SpringExtension.class)
+public class BookRepositoryTest {
+
+    @Autowired
+    private BookRepository bookRepository;
+
+    private Book oneTestBook;
+
+    @Before
+    public void setUp() {
+        oneTestBook = new Book();
+        oneTestBook.setAuthor("Stephen King");
+        oneTestBook.setGenre("Terror");
+        oneTestBook.setImage("https://imagesforus.com/idsarf12.png");
+        oneTestBook.setIsbn("4578-245654");
+        oneTestBook.setPages("1500");
+        oneTestBook.setPublisher("Viking Press");
+        oneTestBook.setSubtitle("Worst Clown Ever");
+        oneTestBook.setTitle("It");
+        oneTestBook.setYear("1986");
+
+    }
+
+    @Test
+    public void whenCreateBook_thenBookIsPersisted() {
+        Book persistedBook = bookRepository.findByTitle("It")
+                .stream().findFirst().orElse(new Book());
```

Comment:
You can throw an exception using `orElseThrow` so that the test fails immediately, because the expected result is that the repository returns the book :)